README.md

```diff
@@ -80,6 +80,42 @@ If you find a bug, feel free to [open an issue](https://github.com/quantopian/py
 You can also join our [mailing list](https://groups.google.com/forum/#!forum/pyfolio) or
 our [Gitter channel](https://gitter.im/quantopian/pyfolio).
 
+## Support
+
+Please [open an issue](https://github.com/quantopian/pyfolio/issues/new) for support.
+
+### Deprecated: Data Reading via `pandas-datareader`
+
+As of early 2018, Yahoo Finance has suffered major API breaks with no stable
```

Comment:
Rather than emphasize that the readers are deprecated, I would focus on making pyfolio independent of a benchmark being present, that's an enhancement. Then discuss how empyrical does not provide them anymore so if you need them, you have to find them.

Reply:
If that's the case I'd rather put that in the release notes. I'll remove the README update here.